document/core/intro/overview.rst

```diff
@@ -23,6 +23,13 @@ This language is structured around the following concepts.
   Instead, integers are interpreted by respective operations
   as either unsigned or signed in two’s complement representation.
 
+  In addition to the basic value types above, a single 128 bit wide
+  value type is added to represent different types of packed data.
```

Comment:
Avoid formulations like "is added", since once in the spec, a feature "just is".